### ISSUE TITLE
gtags-auto-update error after enabling configuration layers

### ISSUE THREAD

**Issue by Skyerin** [syl20bnr/spacemacs#2153]
Hi,

on a fresh install of emacs & spacemacs, opening dired (C-x d) works fine and always displays correctly.

However, after enabling some configuration layers (git, markdown, org, shell, auto-completion, javascript, html, php, sql, syntax-checking, themes-megapack & emacs-lisp), I then get the following error when trying to open up dired;

"symbol's value as variable is void: gtags-auto-update"

And this causes dired to not load - I get the small box at the bottom of the screen where I can navigate through the directories, but when you select the directory and it normally opens up large to navigate the files, that doesn't happen.

Thanks

**Comment by skippednote** [comment]
@Skyerin I'm not using spacemacs but I was facing the same issue and landing here.It started since I installed the `php-mode` package. I've remove it and now `C-x d` works just fine. 

Waiting for other to chime in with better solutions.

**Comment by bmag** [comment]
I investigated and found the problem. This is a bug in `emacs-drush.el`, which is part of `drupal-mode` package.

Here is the culprit:
```elisp
(unless (boundp 'gtags-auto-update)
         gtags-auto-update)
```
The code checks if `gtags-auto-update` exists, then tries to access it when it *doesn't* exist...

You can workaround the bug by adding this line to your emacs config, *before* loading `drupal-mode` (I put it inside my `dotspacemacs/config`):
```elisp
(setq gtags-auto-update nil)
```

**Comment by arnested** [comment]
Thanks for catching this. I fixed it in arnested/drupal-mode#61 and made a release 0.6.1 to Marmelade repos. Melpa and Melpa stable will pick this up shortly I guess.

**Comment by Skyerin** [comment]
works fine now. thanks all!
